raster-foundry/raster-foundry
# Add field to users for notification opt-in status

## Overview

This PR adds a field, `email_notifications`, that defaults to false. It also makes the `name` field on `tool-runs` optional so that 

### Checklist

~- [ ] Styleguide updated, if necessary~


## Testing Instructions
 * Make sure you run migrations
 * POST or PUT a tool-run without a name, it should successfully complete
 * A GET to `/api/users/<userid>` should return an object containing an `emailNotifications` field that is `false`
